README.md

```diff
@@ -1,3 +1,8 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+To run this project locally, you must undergo the following steps:
+npm install
+npm install pg
+npm install dotenv
+
+As the database names are hidden, you must create your own databases and link them to the seed file by using .env file(s) - one for the test data and one for the development data
```

Comment:
Do add info on exactly what the files should be called (e.g. .env.test) as if the names aren't what the dotenv package is expecting the database names won't be found